src/Explorer/Tabs/QueryTab/QueryTabComponent.tsx

```diff
@@ -225,6 +226,20 @@ export default class QueryTabComponent extends React.Component<IQueryTabComponen
     }
   };
 
+  public onDownloadQueryClick = (): void => {
+    const text = this.getCurrentEditorQuery();
+    const queryFile = new File([text], `SavedQuery.sql`, { type: "text/plain" });
```

Comment:
Suggest using a .txt file extension. The query language depends on the account API, and the open file from disk dialog defaults to .txt files.

Reply:
Good point. I had it as `txt` originally but then noticed we talk about it as a "SQL Query" else where. I think `txt` does make more sense here though, especially given that we default the open dialog to that.

Reply:
Done!